cobc/pplex.l

```diff
@@ -159,6 +159,7 @@ static int	fill_continued_alnums = 1; /* whether continued alphanumeric
 					      spaces up to text column */
 static int	emit_area_a_tokens = 0;
 
+
```

Comment:
that extra line seems to not have a purpose

Reply:
Probably a leftover from the removed parts which have been moved to #180 ...